developer-tools/scaleway-cli/reference-content/console-keyboard-shortcuts.mdx

```diff
@@ -0,0 +1,33 @@
+---
+meta:
+  title: Scaleway Console CLI keyboard shortcuts
```

Comment:
```suggestion
  title: Scaleway console CLI keyboard shortcuts
```

Reply:
We don't usually capitalize console - unless there is some sort of exception here?